src/main/java/mate/academy/springboot/swagger/controller/ProductController.java

```diff
@@ -0,0 +1,96 @@
+import mate.academy.springboot.swagger.dto.mapper.ProductMapper;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.service.ProductService;
+import mate.academy.springboot.swagger.util.SortUtil;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/products")
+@RequiredArgsConstructor
+public class ProductController {
+    private final ProductMapper productMapper;
+    private final ProductService productService;
+    private final SortUtil sortUtil;
+
+    @GetMapping
+    @ApiOperation(value = "Get all products")
+    public List<ProductResponseDto> findAll(
+            @RequestParam(defaultValue = "20")
+            @ApiParam(value = "default value is 20") Integer count,
+            @RequestParam(defaultValue = "0")
+            @ApiParam(value = "default value is 0") Integer page,
+            @RequestParam(defaultValue = "id")
+            @ApiParam(value = "default value is id") String sortBy) {
+        Sort sort = sortUtil.createSort(sortBy);
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAll(pageRequest).stream()
+                .map(productMapper::toResponseDto)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/by-price")
+    @ApiOperation(value = "Get all products with a price between two values")
+    public List<ProductResponseDto> findAllWherePriceBetween(
+            @RequestParam BigDecimal from,
+            @RequestParam BigDecimal to,
+            @RequestParam(defaultValue = "20")
+            @ApiParam(value = "default value is 20") Integer count,
+            @RequestParam(defaultValue = "0")
+            @ApiParam(value = "default value is 0") Integer page,
+            @RequestParam(defaultValue = "id")
+            @ApiParam(value = "default value is id") String sortBy) {
+        Sort sort = sortUtil.createSort(sortBy);
+        PageRequest pageRequest = PageRequest.of(page, count, sort);
+        return productService.findAllByPriceBetween(from, to, pageRequest).stream()
+                .map(productMapper::toResponseDto)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    @ApiOperation(value = "Create a new product")
+    public ProductResponseDto create(@RequestBody ProductRequestDto requestDto) {
```

Comment:
Add description for request body https://stackoverflow.com/questions/64645528/java-spring-boot-openapi-3-how-to-add-description-for-requestbody